docs/application/dotnet/guides/app-management/application-launcher.md

```diff
@@ -0,0 +1,304 @@
+            background.Size2D = new Size2D(appWindow.Size.Width, appWindow.Size.Height);
+            background.Position2D = new Position2D(0, 0);
+            appWindow.GetDefaultLayer().Add(background);
+        }
+```
+
+In next step grid component for application icons is created. Grid spacing and columns number is defined in the [GridLayout](/application/dotnet/guides/nui/grid-layout.md) object. The `appGrid` component width and height is set to fill its parent:
+
+```csharp
+            View appGrid = new View()
+            {
+                WidthResizePolicy = ResizePolicyType.FillToParent,
+                HeightResizePolicy = ResizePolicyType.FillToParent,
+                Layout = new GridLayout()
+                {
+                    Columns = 8,
+                    GridOrientation = GridLayout.Orientation.Horizontal,
+                    RowSpacing = 35f,
+                    ColumnSpacing = 12f,
+                    Padding = new Extents(30, 30, 30, 30)
+                }
+            };
+```
+The [PackageManager.GetPackages()](/package-manager.md) is used to obtain all installed packages. Each package may contains several applications. Because of that `pkg.GetApplications()` is used. `ApplicationInfo` object is used to filter apps which should be displayed. In this case `NUIApplicationLauncher`, app with no icon or app with `IsNoDisplay` parameter will not be inserted into `appGrid`:
+
+```csharp
+            IEnumerable<Package> packageList = PackageManager.GetPackages();
+
+            foreach (var pkg in packageList)
+            {
+                var list = pkg.GetApplications();
+
+                foreach (var app in list)
+                {
+                    if (!app.IsNoDisplay && app.IconPath != null && app.Label != "NUIApplicationLauncher") {
+                        var icon = new ApplicationIcon(app.Label, app.IconPath, new Size2D((), 139), app.ApplicationId);
+                        icon.ApplicationIconClicked += OnAppIconClicked;
+                        appGrid.Add(icon);
+                    }
+                }
+            }
+```
+Last step is to insert `appGrid` object in the application window:
+
+```csharp
+    appWindow.GetDefaultLayer().Add(appGrid);
+```
+
+Clicked event setups `AppLauncher` and pass it to [AppControl](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.AppControl.html) `SendLaunchRequest()` API. Now
+selected applications starts:
+
+```csharp
+        public void OnAppIconClicked(object sender, ApplicationIconClickedEventArgs args)
+        {
+            AppLauncher.ApplicationId = args.AppId;
+            AppLauncher.Operation = AppControlOperations.Default;
+            AppControl.SendLaunchRequest(AppLauncher);
+        }
+```
+
+Common method to handle `back` button pressed event. In this case application exits, but normally the launcher app should don't call `Exit()` API:
```

Comment:
Please recheck and correct this sentence

Reply:
How about: '[Step number]. The `back` button clicked event handler implementation. ?